services/login/src/handlers/mozilla-auth0.js

```diff
@@ -162,7 +145,9 @@ class Handler {
       encodedUserId = encodedUserId.replace(/\|[^|]*$/, '');
     }
 
-    return decode(encodedUserId);
+    const userId = decode(encodedUserId);
+
+    return this.getUser({userId, checkIfActive: false});
```

Comment:
Nice job spotting and replicating the existing behavior, but I think this should be `checkIfActive: true`. The clientId -> user translation is used in `scanner.js` when checking whether existing clients have more scopes than their corresponding user does. It seems that process does not consider whether the user is inactive!

(and then it makes sense to drop the `checkIfActive` argument)